simplesat/rules_generator.py

```diff
@@ -1,9 +1,8 @@
 import collections
 import enum
 
-from enstaller.errors import EnstallerException
-
 from .constraints import Requirement
+from .errors import NoPackageFound, SolverException
```

Comment:
Once `sat-salver` does not depend on `enstaller` anymore, I will move the legacy solver in `sat-solver` as well, so there will be no duplicate. Until then, there is a bit of a chicken and egg situation :)